### ISSUE TITLE
Telegram Bot not working

### ISSUE THREAD

**Issue by lasmaty07** [aetaric/checkrr#50]
## **Expected behavior**
get a notification via telegram to the user set on username, when I start a run. 

## **Actual behavior**
I'm not getting any notifications. ever.

## **Information about the Issue**

my docker-compose.yaml
```yaml
  checkrr:
    container_name: checkrr
    image: aetaric/checkrr:latest
    volumes:
      - ${USERDIR}/docker/checkrr/config/checkrr.yaml:/etc/checkrr.yaml
      - ${USERDIR}/docker/checkrr/config/checkrr.db:/checkrr.db
      - /media/wdblue/movies:/media/movies
      - /media/wdblue/tvmedia/tv
    ports:
      - 8585:8585
    environment:
      - PUID= ${PUID}
      - PGID= ${PGID}
      - TZ=${TZ}
```

this is my `checkrr.yaml`:

```yaml
notifications:
  telegram:
    token: "bot_token"
    username: "@my_username"
    notificationtypes:
      - reacquire
      - unknowndetected
      - startrun
      - endrun
```

I dont see anything in `checkrr.log` names telegram

the token is working on sonarr and radarr, also testing with curl

##  **Steps to reproduce the behavior**

- docker-compose up -d
- http://127.0.0.1:8585
- click on "run now"

**Comment by douginoz** [comment]
For what it's worth, I don't get anything in pushover either.  Perhaps notifications isn't enabled yet.

**Comment by aetaric** [comment]
Will be patched in 3.1.2. 

For your reference and for reference of anybody tracking this, make sure you've at least started the conversation with the bot you create via BotFather.
